File: clients/feigram-movil/app/src/main/java/com/example/feigram/network/model/posts/Comment.kt
package com.example.feigram.network.model.posts

import com.google.gson.annotations.SerializedName

data class Comment(
    @SerializedName("post_id")
    val postId: String,

    @SerializedName("user_id")
    val userId: String,

    @SerializedName("text_comment")
    val textComment: String,

    @SerializedName("created_at")
    val createdAt: String
)

File: clients/feigram-web/app/DTO/Comment.cs
namespace app.DTO
{
    public class Comment
    {
        public string post_id { get; set; } = string.Empty;
        public string user_id { get; set; } = string.Empty;
        public string text_comment { get; set; } = string.Empty;
        public DateTime created_at { get; set; }
    }
}